bindings/wasm/src/account/update/mod.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+mod attach_method_relationship;
+mod create_method;
+mod create_service;
+mod delete_method;
+mod delete_service;
+mod detach_method_relationship;
```

Comment:
Aside: still slightly annoyed that `attach_method_relationship` and `detach_method_relationship` are inconsistently pluralised.